### DISCUSSION TITLE
Installation question

### DISCUSSION THREAD

**Discussion by InaudibleSound** [AloneLiberty/FlipperNestedRecovery#18] [Q&A]
Hi AloneLiberty,

I just came across your app, and used the Nested Attack, unfortunately i can't seem to solve the install process with the Nested Recovery. I've installed Python, but can't figure out what and how i have to install the Recovery program. If you are still active, would you be able to guide me how to do so. Thank you

**Comment by AloneLiberty** [comment]
There are Python 3.12 builds now. You can follow guide: https://github.com/AloneLiberty/FlipperNested/wiki/Usage-guide
